Mathlib/Topology/UniformSpace/CompactConvergence.lean

```diff
@@ -318,8 +325,43 @@ theorem tendsto_iff_tendstoUniformly :
   rw [tendsto_iff_forall_isCompact_tendstoUniformlyOn, ← tendstoUniformlyOn_univ]
   exact ⟨fun h => h univ isCompact_univ, fun h K _hK => h.mono (subset_univ K)⟩
 
+open UniformFun in
+/-- When `α` is compact, `f : X → C(α, β)` is continuous if any only if it is continuous when
+reinterpreted as a map `f : X → α →ᵤ β`. -/
+theorem continuous_iff_continuous_uniformFun {X : Type*} [TopologicalSpace X] (f : X → C(α, β)) :
+    Continuous f ↔ Continuous (fun x ↦ ofFun (f x)) := by
+  rw [continuous_iff_continuous_uniformOnFun]
+  exact UniformOnFun.uniformEquivUniformFun β _ isCompact_univ
+    |>.isUniformInducing.isInducing.continuous_iff
```

Comment:
Could you also add the `IsUniformEmbedding` statement? I think it can be useful.